### DISCUSSION TITLE
Disable on device stitching for GoPro Max

### DISCUSSION THREAD

**Discussion by eneerhut** [gopro/labs#722] [Feature Requests]
It would be great to have an option to disable on device stitching of the GoPro Max images. The GoPro Max is a useful tool for map data/street-level imagery collection, and 3D reconstructions of the scene work better with the raw images. [Here is an example](https://www.mapillary.com/app/?lat=37.781605555556&lng=-122.4205&z=17&pKey=2786196298182537&focus=photo&x=0.48433699611001546&y=0.4926699857506876&zoom=0) of the GoPro Max being used for street-level imagery collection.

If disabling stitching also permits a higher frame rate in timelapse photo mode e.g. 2 FPS, that would be another benefit.

**Comment by dnewman-gpsw** [comment]
It is not something Labs can do, and Labs doesn't add new video or photo modes. The support for dual fisheye is not currently in the camera, even internally, yes I know that seems odd.  One reason is the EAC encoding for video means there is less computation in camera to do the stitch, than leave it unstitched.  Encoding two fisheye images at ~3000x3000 (GoPro Fusion style) is more work than two stitched EAC strips of 4096x1344 (MAX) as the compression is more efficient with encoding only active pixels with less distortion.  Photos technically could be dual fisheyes, but for dev reason it shares a lot with video for the stitch.

**Comment by eneerhut** [comment]
Thanks for the quick response and clarification @dnewman-gpsw! It'll be interesting to see what the approach is with the hopefully upcoming GoPro 360º camera.

**Comment by tomaszpieczykolan** [comment]
Something I noticed is that the LRV files are in fact not stitched, however they have a low resolution (1408x704 px). Still, this information might be useful for someone who's looking for two separate videos. You can use ffmpeg or an app like LosslessCut to extract the MP4 video from a LRV file.
